drivers/wifi/nrf700x/osal/fw_if/umac_if/src/fmac_api_common.c

```diff
@@ -411,7 +411,8 @@ enum wifi_nrf_status wifi_nrf_fmac_otp_mac_addr_get(struct wifi_nrf_fmac_dev_ctx
 				      otp_mac_addr,
 				      NRF_WIFI_ETH_ADDR_LEN);
 
-		if (!nrf_wifi_utils_is_mac_addr_valid((const char *)mac_addr)) {
+		if (!nrf_wifi_utils_is_mac_addr_valid(fmac_dev_ctx->fpriv->opriv,
```

Comment:
Use the same suggested commit log from earlier commit. And also same argument for `strlen` holds true here.

Reply:
Commit log updated. 
Inclusion of Path in Makefile leads to oher build issue.

Reply:
Can you please share the issue? We should be able to fix this easily by appending to the include path.

Reply:
Offline discussion:

The same C headers are defined in Linux kernel as well, so, if we fix the include path to include `string.h` the automatically the `stddef.h` from kernel is picked up which doesn't have `size_t` defined and causes build issues.

Also, using C standard functions may not be enough, as Linux kernel leverages processor architecture specific features to re-implement C standard functions efficiently, e.g., `memcmp`, so, its better abstract these and let shim implement as per the OS.